class/defaults.yml

```
@@ -211,7 +228,9 @@ parameters:
         existingSecret: ${keycloak:database:secretname}
         image:
           registry: quay.io
+        # master renamed to primary later https://github.com/bitnami/charts/commit/7eabc85fd4fae43127228a22829c7ce3fe85c389
         master:
+          annotations: ${keycloak:_enable_k8up_postgres_annotations:_${keycloak:k8up:enabled}}
```

I don't think there's any harm in having the backup command annotation present, even if the K8up schedule is not deployed.

I'd suggest just having the annotation here and removing the lookup table completely.

```suggestion
          annotations:
            k8up.syn.tools/backupcommand: sh -c 'PGDATABASE="$POSTGRES_DB" PGUSER="$POSTGRES_USER" PGPASSWORD="$POSTGRES_PASSWORD" pg_dump'
```